pkg/controller/clusterinstallation/mattermost.go

```diff
@@ -34,10 +34,21 @@ func (r *ReconcileClusterInstallation) checkMattermostIngress(mattermost *matter
 }
 
 func (r *ReconcileClusterInstallation) checkMattermostDeployment(mattermost *mattermostv1alpha1.ClusterInstallation, reqLogger logr.Logger) error {
-	dbPassword, err := r.getMySQLSecrets(mattermost, reqLogger)
-	if err != nil {
-		return errors.Wrap(err, "Error getting the database password.")
+	externalDB := false
+	var dbPassword string
+	var dbUser string
```

Comment:
nit: you can save a bit of space when you have like types. These two lines could become:
```suggestion
	var dbUser, dbPassword string
```